EFI/OC/config.plist

```diff
@@ -666,25 +666,65 @@
 		<dict>
 			<key>PciRoot(0x0)/Pci(0x1f,0x3)</key>
 			<dict>
+				<key>AAPL,slot-name</key>
+				<string>Internal@0,31,3</string>
+				<key>device-id</key>
+				<data>
```

Comment:
Why adding devices this way? There is no advice for this in Dortania manual.

Reply:
Sorry late answer. Covid-19 related problems at work.

Also Hackintosh app advice. It's just "spoofing" a legit audio device. Can't really say if it is necessary or not (cuz it works without), but doesn't break anything.